sw/device/silicon_creator/rom/e2e/epmp_init_test.c

```diff
@@ -25,7 +25,8 @@ bool test_main(void) {
   dif_lc_ctrl_t lc_ctrl;
   mmio_region_t lc_reg = mmio_region_from_addr(TOP_EARLGREY_LC_CTRL_BASE_ADDR);
   CHECK_DIF_OK(dif_lc_ctrl_init(lc_reg, &lc_ctrl));
-  const bool kDebugEnabled = lc_ctrl_testutils_debug_func_enabled(&lc_ctrl);
+  bool kDebugEnabled;
```

Comment:
Nit: we only use the [naming convention ](https://google.github.io/styleguide/cppguide.html#Constant_Names) of a "k" followed by mixed case for constants. Since this is no longer a constant, you want something like:
```suggestion
  bool debug_enabled = false;
```